esphome/components/mitsubishi_itp/mitsubishi_uart-packetprocessing.cpp

```diff
@@ -355,12 +384,106 @@ void MitsubishiUART::process_packet(const RemoteTemperatureSetRequestPacket &pac
 
     publish_on_update_ |= (old_thermostat_temp != thermostat_temperature_sensor_->raw_state);
   }
-};
+}
+
+void MitsubishiUART::process_packet(const ThermostatSensorStatusPacket &packet) {
+  if (!enhanced_mhk_support_) {
+    route_packet_(packet);
+    return;
+  };
+
+  ESP_LOGV(TAG, "Processing inbound %s", packet.to_string().c_str());
```

Comment:
Move this log line to top of function so we know we got one (even if we're not doing enhanced).  Same with following functions.

Reply:
Splitting the difference and giving passthrough its own log entry.

Reply:
That doesn't match how passthroughs are handled anywhere else.  I'm okay with adding an additional line that says we're passing through (this is verbose after all, no need to be frugal) though, or if we *must* differentiate let's do it the same across all packets.